### DISCUSSION TITLE
アイコン未設定時のデフォルト画像

### DISCUSSION THREAD

**Discussion by AyumuAkimoto** [cuculus-dev/.github#5] [Ideas]
### 要望/アイデアの概要

アカウント作成時に未設定の場合に表示する画像をランダムで色などを変えて生成できるようにしたい。
卵アイコンで背景と卵の色だったり模様がついたりするやつ
ある程度元となる画像データにバックエンド側で色などをランダム生成させる。

### 利点

- 卵アイコンが懐かしい
- 初期設定でもタイムラインがにぎやかせる

**Comment by takecchi** [comment]
調べてみたらホトトギスの卵って茶色らしい(?)
卵ってわかりづらいから基本白でいいと思うけど、レア要素で色が変わったり柄が入るの面白そう
